### DISCUSSION TITLE
Quarkus create native build from ARM aarch64 to 64-bit AMD

### DISCUSSION THREAD

**Discussion by icdcdev** [quarkusio/quarkus#31165] [Q&A]
Hi Community!
I was wondering if its possible to create a native runner from Macbook M1 with aarch64 architecture to 64-bit AMD architecture. I tried and I get the following response:

This may be caused by building the native binary in a Linux container while the host is macOS

Thanks!

**Comment by cescoffier** [comment]
Unfortunately that's not possible except when using emulation (qemu).
